### PR TITLE
2022.1/universal/backport/bugfix case 1392965 max light count mismatch

### PR DESCRIPTION
**Pull request by eh-unity**
---
### Purpose of this PR
Back port of https://github.com/Unity-Technologies/Graphics/pull/6837

---
### Testing status
None.

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

**Review by phi-lira — APPROVED** [review]

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 Version Updated
 The version number for this package has increased due to a version update of a related graphics package.
```
Comment (ellioman) [inline com.unity.render-pipelines.universal/CHANGELOG.md:11]: Same here as in the other backport. You can remove those lines :)

**Review by ellioman — APPROVED** [review]
Good. Just need to fix the changelog.

File: com.unity.render-pipelines.universal/CHANGELOG.md (hunk 1 above)
Comment (eh-unity) [reply com.unity.render-pipelines.universal/CHANGELOG.md]: done.

**Comment by eh-unity** [comment]
Looks like the same errors are in 22.1/staging. The seem like RTHandle bugs (missing backports?).

**Comment by eh-unity** [comment]
Failures appear to be the same as in 22.1/staging and some test timeouts.

**Pull request merged**